content/en/monitors/types/watchdog.md

```diff
@@ -31,43 +31,41 @@ In this section, choose between an APM, Infrastructure, or Logs alert:
 
 An APM alert is created when Watchdog detects anomalous behavior on your system's services or their child resources.
 
+{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-apm.png" alt="Configuring a Watchdog Monitor on APM" style="width:55%;">}}
+
 ### Select sources {#select-sources-1}
 
-Choose your [primary tags][1], [service][2], and [resource][3] from the dropdown menus.
+Select the scope to be alerted on by configuring:
+
+* The type of Watchdog anomaly: Error Rates, Latency, Hits, or any of them
+* The value for APM primary tags ([instruction to configure APM primary and second primary tags][1])
+* The [APM service][2] (choose `Any services` to monitor them all)
+* The [APM resource][3] (choose `*` to monitor them all)
+* The dimensions you want to [group notifications by][4]
 
-After your selections are made, the graph at the top of the monitor creation page displays the matching Watchdog events over time, along with a list of events.
+After your selections are made, the graph at the top of the monitor creation page displays the matching Watchdog events over time.
 
 [1]: /tracing/guide/setting_primary_tags_to_scope/#environment
 [2]: /tracing/services/service_page/
 [3]: /tracing/services/resource_page/
+[4]: /monitors/configuration/?tab=thresholdalert#alert-grouping
 {{% /tab %}}
 {{% tab "Infrastructure" %}}
 
-Infrastructure-wide alerts can include issues over the following integrations:
-
-* [System][1]: Host-level memory usage (memory leak), TCP retransmit rate, etc.
-* [Redis][2]
-* [PostgreSQL][3]
-* [NGINX][4]
-* [Amazon Web Services][5]: For the [S3][6], [ELB/ALB/NLB][7], [CloudFront][8], and [DynamoDB][9] Amazon services.
-
-After selecting Infrastructure, the graph at the top of the monitor creation page displays Watchdog events over time, along with a list of events.
+{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-infra.png" alt="Configuring a Watchdog Monitor on Infrastructure" style="width:55%;">}}
 
 ### Select sources {#select-sources-2}
 
-No selection is necessary. You are notified when Watchdog detects issues across your infrastructure.
+Select the scope to be alerted on by configuring:
 
+* The Infrastructure integration to cover (select `Any Infrastructure alert` to cover them all): [full list of covered integrations by Watchdog infrastructure][1])
+* The Tags available for the selected integration
```

Comment:
I'm not sure which part of the UI this is referencing, could you please clarify? In the demo account, I can see only the options to select the integration and configure the dimensions to group notifications by.